src/frontend/screens/Observation/index.tsx

```diff
@@ -86,15 +97,16 @@ export const ObservationScreen: NativeNavigationComponent<'Observation'> = ({
               <Text style={styles.textNotes}>{observation.tags.notes}</Text>
             </View>
           ) : null}
-
-          {/* {!!photos.length && (
-            <ThumbnailScrollView
-              photos={
-                photos
-              }
-              onPressPhoto={() => {}}
-            />
-          )} */}
+          <ThumbnailScrollView
```

Comment:
Should we show this view if there are no photos?

Reply:
This will return empty scroll view which AFAIK renders as "nothing" if it has no items but I'll change in nonetheless to keep previous (commented out) behavior.